### ISSUE TITLE
font required for pkg

### ISSUE THREAD

**Issue by PythonCoderUnicorn** [delabj/ggCyberPunk#11]
Hello,

I am testing out your pkg and really like it, thanks for making it.

My issues are:
- the ``Aldrich`` font is not loaded and gave me 50 warnings but was fixed once i loaded ``showtext`` pkg
```
library(showtext)
font_add_google(family = 'Aldrich', 'Aldrich')
showtext_auto()
```
- 2nd issue is when using your example code for the ``scale_color_linesaber(reverse = T)`` i get an error saying that this function can't be found and when i check for it with ``ggCyberPunk::`` it is not there.

```
# this fails to work for function "scale_color_linesaber"
df2 %>%
  ggplot(aes(x=Time, y = count, color = group, fill= group))+
  geom_linesaber(alpha = 1, size = 1, glow_alpha = 0.03)+
  # ggCyberPunk::geom_linesaber()+
  theme_cyberpunk()+
  scale_color_linesaber(reverse = T)+
  ggtitle("geom_linesaber()", subtitle = "From ggCyberPunk")
```

this works
```
df2 %>%
  ggplot(aes(x=Time, y = count, color = group, fill= group))+
  geom_linesaber(alpha = 1, size = 1, glow_alpha = 0.03)+
  ggCyberPunk::geom_linesaber()+
  theme_cyberpunk()+
  # scale_color_linesaber(reverse = T)+
  ggtitle("geom_linesaber()", subtitle = "From ggCyberPunk")
```
- 3rd issue is regarding the ``theme_delabj_dark()`` function with error ``could not find function "theme_delabj_dark" `` but works when i use ``ggdark::dark_mode()``

this works 
```
df3 %>%
  ggplot(aes(x=I, y = count, color = group, fill= group))+
  geom_glowing_area(glow_size = .5, glow_alpha = 0.05,layers = 10)+
  # theme_delabj_dark()+
  ggdark::dark_mode()+
  scale_fill_manual(values = c("#08F7FE", "#FE53BB"))+
  scale_color_manual(values = c("#08F7FE", "#FE53BB"))+
  labs(title = "geom_glowing_area")
```
 plot
![image](https://user-images.githubusercontent.com/55933131/141362991-0224a4c9-c8ca-469c-9676-1a9976fe5523.png)

**Comment by delabj** [comment]
Hey there! 

1) I think I wrote this package before I knew about showtext, so the support might not work as it is right now. I do think that showtext is more useful than extra font which has brought OS specific issues. The warnings are from ggplot2 proper because it can't find my default font. I'm making a seperate issue (#12) for my fixes here. 


2) It looks like when I last changed my docs I forgot to export the functions. It's still there with ggcyberpunk:::, but this should be an easy fix. 

3) That is an issue of me using my personal package too often without checking. I'll get that fixed when I fix 2. I've opened an issue in #13
